.github/workflows/java8_integration_tests_ft.yml

```diff
@@ -53,6 +53,7 @@ jobs:
           ALLUXIO_DOCKER_FORK_COUNT=1 \
           ALLUXIO_DOCKER_NO_TTY=true \
           ALLUXIO_DOCKER_GIT_CLEAN=true \
+          ALLUXIO_DOCKER_MVN_PROJECT_LIST_TEST=dora/tests/integration \
```

Comment:
this file is missing the `ALLUXIO_DOCKER_MVN_PROJECT_LIST_COMPILE`

Reply:
Yes, it is in our expectation. Maybe it takes more time, but not introducing a worse result. I will try to specify the `ALLUXIO_DOCKER_MVN_PROJECT_LIST_COMPILE`, if no bugs are introduced.